package.json

```diff
@@ -248,6 +249,7 @@
     "@commitlint/config-conventional": "^8.1.0",
     "@commitlint/travis-cli": "^8.1.0",
     "@ngageoint/closure-webpack-plugin": "^1.0.0",
+    "@ngageoint/istanbul-instrumenter-loader": "^1.0.0",
```

Comment:
Is this a new repository/npm package for us to manage?

Reply:
Yes. The original is linked in the description and is archived/no longer managed. Code-wise it's extremely small and shouldn't change, we just need to update a dependency (`istanbul-lib-instrument`) to fix an issue with classes and Angular.